ui/src/funding/pool.ts

```diff
@@ -0,0 +1,335 @@
+import * as svelteStore from "svelte/store";
+
+import * as contract from "./contract";
+import * as transaction from "../transaction";
+import * as validation from "../validation";
+
+import { Wallet, Account, Status as WalletStatus } from "../wallet";
+import * as remote from "../remote";
+import { BigNumber, BigNumberish, ContractTransaction, ethers } from "ethers";
+
+export const store = svelteStore.writable<Pool | null>(null);
+
+export interface Pool {
+  data: remote.Store<PoolData>;
+
+  // Get the account that owns this pool. Should be the connected wallet account.
```

Comment:
> Should be the connected wallet account.
	
What does this mean? How does this influence the return value or potential exceptions?

Reply:
The comment itself does not influence the return value, it just informs the reader that the account owning the pool instance we have at hand is (that is, "should be") the connected wallet account. So doing `pool.getAccount` should be the same as doing `$wallet.connected.account` (when the wallet is connected, of course).

Let me know how this can be clearer :+1:

Reply:
I've cut the second sentence out, let me have your thoughts.

Reply:
Fine with me.

Just to clarify where the confusion came from: It was not clear what the subject of the sentence was and who would be responsible for ensuring the requirement and how it would influence anything regarding the method.